README.md

```diff
@@ -0,0 +1,13 @@
+# zlib [![CI][CI badge]][CI page] [![Hackage][Hackage badge]][Hackage page]
+
+**Compression and decompression in the gzip and zlib format**
+
+This package provides a pure interface for compressing and decompressing streams of data represented as lazy ByteStrings.
+It uses the zlib C library so it has high performance. It supports the "zlib", "gzip" and "raw" compression formats.
```

Comment:
```suggestion
This package provides a pure interface for compressing and decompressing streams of data represented as lazy `ByteString`s.
It uses the `zlib` C library so it has high performance. It supports the `zlib`, `gzip` and `raw` compression formats.
```
Trivial changes attract most suggestions ;)

Reply:
I literally picked this up from the hackage homepage from zlib ;) Should I change that in the Cabal file as well?